www/includes/easyparliament/templates/html/_stripe_donate.php

```diff
@@ -0,0 +1,23 @@
+
+<div class="donation-box">
+
+
+  <h2>Donate to TheyWorkForYou</h2>
+
+
+
+  <a class="button button--red" href="https://buy.stripe.com/6oE6sddMc15Dd7qaEH">£10 Annual donation</a>
+  <a class="button button--red" href="https://donate.stripe.com/5kA6sddMcaGd0kEfYZ?&__prefilled_amount=2000">£20 One-off donation</a>
+  <a class="button button--red" href="https://donate.stripe.com/5kA6sddMcaGd0kEfYZ">Custom donation</a>
+  
+<div class="payment-options">
+<span class="payment-options-text">Payment options:</span>
+<img role="presentation" src="/images/payment/amex-b933c9009eeaf8cfd07e789c549b8c57.svg" alt="amex" id="amex">
+<img role="presentation" src="/images/payment/mastercard-86e9a2b929496a34918767093c470935.svg" alt="mastercard" id="mastercard">
+<img role="presentation" src="/images/payment/visa-fb36094822f73d7bc581f6c0bad1c201.svg" alt="visa" id="visa">
+<img role="presentation" src="/images/payment/google_pay-ca6cc2f4ee364c7966f8fabf064849fe.svg" alt="GOOGLE_PAY" id="GOOGLE_PAY">
```

Comment:
Seems odd to have this and not apple pay when stripe supports both? Do we really want to show card mechanisms here?

Reply:
I'm not that invested in it - it's simpler without - was just experimenting replicating what the stripe box had done previously. 

Only thing is being clear what it does take so people don't need to click through to the main page but also... it's fine if they do that.